ci/properties/dotnet-core-desktop.properties.json

```diff
@@ -0,0 +1,6 @@
+{
+    "name": ".NET Core Desktop",
+    "description": "Build, test, sign and publish a desktop application built on .NET Core.",
+    "iconName": "dotnetcore",
+    "categories": ["C#", "Visual Basic", "WPF", ".NET", "WindowsForms"]
```

Comment:
These categories need to align to the languages that we match on with linguist.  @mscoutermarsh do you know this dark magic?  If not do you know who does?

Reply:
We'd really like to keep some version of "WindowsForms" in there since it is a kind of desktop app, similar to WPF.   But if there isn't anything that maps to it, we can let it go.  Please advise.

Reply:
I'm going to remove "WindowsForms" to help this PR move.  We can always add it another day.